buildcomponents.yaml

```diff
@@ -0,0 +1,13 @@
+.migrate:
```

Comment:
The only components in this buildcomponents file are database migrate and grunt commerce deploy prototypes.

"deploy" components would be better than "build"